charts/kube-prometheus-stack/templates/prometheus/prometheus.yaml

```diff
@@ -32,7 +32,7 @@ spec:
 {{ toYaml .Values.prometheus.prometheusSpec.apiserverConfig | indent 4}}
 {{- end }}
 {{- if .Values.prometheus.prometheusSpec.image }}
-  baseImage: {{ .Values.prometheus.prometheusSpec.image.repository }}
+  image: {{ .Values.prometheus.prometheusSpec.image.repository }}
```

Comment:
Unlike `baseImage`, this needs the `tag` appended. (Version is still required as part of the spec)

```
image: {{ .Values.prometheus.prometheusSpec.image.repository }}:{{ .Values.prometheus.prometheusSpec.image.tag }}
```

Reply:
Thanks for catching this. Fixed!